scripts/tools/flip_video.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+"""
+This script helps to flip videos horizontally to create videos of opposite side.
+
+Usage:
+  flip_video.py --path_in=PATH_IN --path_out=PATH_OUT
+  flip_video.py (-h | --help)
+
+Options:
+  --path_in=PATH_IN     Full path to the videos folder
+  --path_out=PATH_OUT   Full path to save flipped videos
+"""
+
+import ffmpeg
+from docopt import docopt
+from pathlib import Path
+import os
+
+if __name__ == '__main__':
+    args = docopt(__doc__)
+    videos_path_in = Path(args['--path_in'])
+    videos_path_out = Path(args['--path_out'])
+
+    video_suffix = '.mp4'
+
+    os.makedirs(videos_path_out, exist_ok=True)
+    for directory in videos_path_in.iterdir():
+
+        print(f'Processing videos in folder: {directory.name}')
+        output_folder = videos_path_out / directory.name
+        os.makedirs(output_folder, exist_ok=True)
+
+        for video in directory.iterdir():
+            print(f'Processing video: {video.name}')
+
+            original_video_name = video.stem
+            flipped_video_name = original_video_name + '_flipped' + video_suffix
```

Comment:
Need suggestion on naming of output video.

Reply:
This works. Could combine with the line above instead of setting a new variable for `original_video_name`